src/main/java/technology/semi/weaviate/client/v1/graphql/GraphQL.java

```diff
@@ -47,6 +48,10 @@ public Get get() {
     return new Get(config);
   }
 
+  public RawGraphQL raw() {
```

Comment:
I would change the implementation of this method just to build the query inside and return the `run` method, example:
```
client.graphQL().raw("{Get{Pizza{_additional{id}}}}").run();
```